tracing/tracingtest/mocktracer.go

```diff
@@ -13,6 +13,8 @@ type MockTracer struct {
 	spans      atomic.Int32
 }
 
+type MockSpan = mocktracer.MockSpan
```

Comment:
This type alias is to not require caller to import "github.com/opentracing/opentracing-go/mocktracer".